src/scss/screens/home/components/_search.scss

```diff
@@ -31,6 +31,13 @@
     padding-left: 60px;
   }
 
+  &__select,
```

Comment:
never add static width to elements, check this again and use flex property for flex items